core/src/impl/Kokkos_HostThreadTeam.hpp

```diff
@@ -91,9 +91,15 @@ class HostThreadTeamData {
   //   [ thread_local ]     = [ m_thread_local    .. m_scratch_size )
 
   enum : int { m_pool_members = 0 };
-  enum : int { m_pool_rendezvous = m_pool_members + max_pool_members };
-  enum : int { m_team_rendezvous = m_pool_rendezvous + max_pool_rendezvous };
-  enum : int { m_pool_reduce = m_team_rendezvous + max_team_rendezvous };
+  enum : int {
+    m_pool_rendezvous = int(m_pool_members) + int(max_pool_members)
```

Comment:
Do we want function style casts or static_casts here?

Reply:
Thanks for the review, I'll make the change to static_casts and update the PR

Reply:
Replaced the function style casts with static_casts in 9863ac9